load-balancer/api-docs/f5-api-reference/f5-load-balancer.rst

```diff
@@ -1446,6 +1605,148 @@ Response
         ]
     }
 
+Disable Pool Member For Maintenance
+-----------------------------------
+
+This setting allows the Pool Member (combination of IP and Port) to accept only new connections that match an existing persistence session.
+Use this feature to prevent new connections to a Pool Member without affecting existing client connections on the same Pool Member.
+
+To monitor connection stats of a Pool Member, see: `Retrieve statistics for pool members`_. Review the first object in the data array. The `serverside` object shows stats on activity to the member. 
+
+To re-enable the Pool Member, see: `Enable Pool Member For Maintenance`_.
+
+*Note: It is important to understand differences between* `Disable vs Offline`_.
+
+:: 
+
+    PUT /pools/{pool_ID}/members/{member_ID}
+
+Request body
+^^^^^^^^^^^^
+
+::
+
+    {
+        "session": "user-disabled"
```

Comment:
put payload for internal raml docs looks bit different. please validate 
https://pages.github.rackspace.com/automationIntegration/bpi.api.python.v2/f5-loadbalancer.html#pools__poolid__members__memberid__put

Reply:
ignore it, was able to find in internally updated raml doc attached in jira.